src/components/info-tile/examples/info-tile-loading.tsx

```diff
@@ -0,0 +1,53 @@
+import { Component, h, State } from '@stencil/core';
+/**
+ * Loading state
+ *
+ * Sometimes the value needs to be calculated, updated, or fetched
+ * through a process that requires some time. In such cases, it is
+ * a great idea to let the users know that the data is being updated.
+ *
+ * To do so, set the `loading` property to `true`. The component will then
+ * show an indeterminate progressbar indicating the data is being updated,
+ * while the older value is still being displayed.
+ *
+ * :::note
+ * Note that this does _not_ disable the link, and most probably you
+ * do not need to it to be disabled neither.
```

Comment:
I had to look this up to be sure, but after a "not", "either" is the correct word to use, rather than "neither" 😊

```suggestion
 * do not need it to be disabled either.
```

Reply:
😳 I blame my English teacher 😄